packages/tools/create-kadena-app/templates/angular/src/app/app.component.html

```diff
@@ -0,0 +1,36 @@
+<div>
+ <main class="grid">
+   <div class="card">
+     <div class="header">
+       <h1 class="title">Welcome to <span>Kadena!</span></h1>
+     </div>
+     <p>This is the Kadena starter template using nextjs to help you get started on your blockchain development. Use the form below
+       to interact with the Kadena blockchain using <code>@kadena/client</code> and edit <code>src/app/app.component.html</code> to get started.</p>
+   </div>
+   <div class="card">
+     <h3 class="cardTitle">Interact with the blockchain</h3>
+     <section class="cardSection">
+       <h4>My Account</h4>
+       <input id="account" [(ngModel)]="account" placeholder="Please enter a valid k:account" />
+     </section>
+     <section class="cardSection">
+      <h4>Write Message</h4>
+       <textarea id="messageToWrite" [(ngModel)]="messageToWrite" [disabled]="account === '' || writeInProgress"></textarea>
```

Comment:
@jermaine150 here you're using a 2-way binding which is not necessary. I believe the actual scenario is to write into `messageToWrite` when the user types. You can achieve this with `[ngModel]="messageToWrite"` then when you click below, the variable `messageToWrite` will be populated with the user's input.

Reply:
@davyengone like we discussed, `[ngModel]="messageToWrite"` will give one-way binding but only from source to view, while in this case we need it the other way around. 
An alternative would be to use `(ngModelChange)="currentHero.name=$event">` but I would prefer 2-way binding over that.

Reply:
Yes, let's go with that approach. The Reactiveform approach is complicated for such an example